LWCE_Core/Src/XComLongWarCommunityEdition/Classes/LWCE_UITacticalHUD_Radar.uc

```diff
@@ -1,5 +1,43 @@
 class LWCE_UITacticalHUD_Radar extends UITacticalHUD_Radar;
 
+simulated function Show()
+{
+    local LWCE_XGUnit kActiveUnit;
+    local TInventory kInventory;
+
+    if (`BATTLE.m_kDesc.m_bIsTutorial)
+    {
+        return;
+    }
+
+    m_kActiveUnit = XComTacticalController(controllerRef).GetActiveUnit();
+    kActiveUnit = LWCE_XGUnit(m_kActiveUnit);
+
+    if (kActiveUnit == none)
+    {
+        Hide();
+        return;
+    }
+
+    kInventory = kActiveUnit.GetTInventory();
+
+    if (kActiveUnit.HasPerk(`LW_PERK_ID(BioelectricSkin)))
+    {
+        super(UI_FxsPanel).Show();
+        ActivateRadarTracking();
+    }
+    else if (class'XGTacticalGameCoreNativeBase'.static.TInventoryHasItemType(kInventory, 255))
```

Comment:
Can you change `255` to `LW_ITEM_ID(MotionTracker)` please? Everything else looks good.

Reply:
I don't think this is the motion tracker, that's id `222` and we don't want to always enable the hud when someone is carrying one. I actually copied that from `UpdateActiveUnit` below, which uses the same ID. Is it supposed to be some flag that identifies the covert operative?

Reply:
Oh you're right, I just assumed it was the motion tracker, sorry. I checked the Long War patches and that whole section is added by them. For this specifically there's a note:

    //if (class'XComTacticalGameCoreNativeBase'.static.TInventoryHasItemType(m_kActiveUnit.GetCharacter().m_kChar.kInventory,98)) 
    // TODO: REplaced 98 with 0xFF

Item ID 98 is the Alien Trophy, and ID 0x98 is Floater Captive, so I suppose whatever their initial plans for this were, they didn't get carried out. 

I'll probably come back and clean up their dead code at some point, but this looks all good.